types/react-i18next/src/I18nextProvider.d.ts

```diff
@@ -4,9 +4,9 @@ import { i18n } from "i18next";
 // tslint:disable-next-line:interface-name
 export interface I18nextProviderProps {
     i18n: i18n;
-    children: React.ReactElement<any>;
     initialI18nStore?: any;
     initialLanguage?: string;
+    children: React.ReactElement<any>;
```

Comment:
maybe `JSX.Element` is sufficient over here ?

Reply:
Replaced with JSX.Element, thanks